docs/source/tutorial/first_steps.rst

```diff
@@ -22,6 +22,54 @@ that you can use :func:`torch.load` to load a model like:
 More information on PyTorch's model persistence can be found at:
 https://pytorch.org/tutorials/beginner/saving_loading_models.html.
 
+Using Learned Embeddings
+------------------------
+The embeddings learned for entities and relations are useful for link
+prediction in PyKEEN, but also generally useful for other downstream
+machine learning tasks like clustering, regression, and classification.
+
+The embeddings themselves are typically stored in a :class:`pykeen.nn.Embedding`,
+which wraps the :class:`torch.nn.Embedding` with several key values. They
+can be accessed like this:
+
+.. code-block:: python
+
+    from pykeen.pipeline import pipeline
+    result = pipeline(model='TransE', dataset='UMLS')
+    model = result.model
+    entity_embeddings = model.entity_embeddings._embeddings.weight.data
+    relation_embeddings = model.relation_embeddings._embeddings.weight.data
+
+However, the :class:`pykeen.nn.Embedding` inherits from the more generalizable
+:class:`pykeen.nn.RepresentationModule`, which can be used for alternative
+implementations. The new-style way to access the embeddings is now like this:
+
+.. code-block:: python
+
+    entity_embeddings = model.entity_embeddings()
+
+More explicitly:
+
+.. code-block:: python
+
+    entity_embeddings = model.entity_embeddings(indices=None)
+
+If you'd like to only look up certain embeddings, you can use the ``indices`` parameter
+and pass a :class:`torch.LongTensor` with their corresponding indices.
+
+New-style models (e.g., ones inheriting from :class:`pykeen.models.ERModel`) are
```

Comment:
We already have models which have more than one representation, e.g., [TransD](https://github.com/pykeen/pykeen/blob/master/src/pykeen/models/unimodal/trans_d.py), where we refer to one of them (more or less arbitrary) as the entity_embedding, and the other ones have custom names.